package.json

```diff
@@ -42,6 +42,7 @@
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-react": "^7.26.0",
     "eslint-plugin-react-hooks": "^4.2.0",
+    "file-loader": "^6.2.0",
```

Comment:
I don't think this dependency is required. `Create React App` should support loading SVGs by default: https://create-react-app.dev/docs/adding-images-fonts-and-files/